CODEOWNERS

```
@@ -178,6 +178,9 @@ k8s/**/common/bar/identity.yaml @hmcts/Fees
 k8s/**/common/bar/namespace.yaml @hmcts/Fees
 
 # IDAM
+k8s/**/common/idam/idam-api.yaml @hmcts/idam
+k8s/**/common/idam/idam-web-public.yaml @hmcts/idam
+k8s/**/common/idam/idam-web-admin.yaml @hmcts/idam
```

Let it be aat for now as prod would need approval from secops in terms of CVEs.

```suggestion
k8s/aat/common/idam/idam-api.yaml @hmcts/idam
k8s/aat/common/idam/idam-web-public.yaml @hmcts/idam
k8s/aat/common/idam/idam-web-admin.yaml @hmcts/idam
```